City-of-Pori/pori-kada
# PORI-168: Moved news image under the body field

To test:

1. drush fra -y && drush cc all
2. Go to any news node page that has body and image and make sure the image comes after the body field.
